packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md

```diff
@@ -336,7 +334,13 @@ Nimble already has a mechanism for clients to provide custom columns by deriving
 
 ### Additional unit APIs
 
-We considered adding attributes `prefix-field-name` and `suffix-field-name` which would allow those values to be populated with a dynamic field value rather than a constant string. This could help use cases where values have disparate types or units. We decided this is not necessary initially but could revisit in the future. See [this thread](https://github.com/ni/nimble/pull/1268#discussion_r1212385898) for more discussion.
+We considered adding attributes for `prefix` and `suffix` that could be used to specify any string to append to the front or back of the formatted number. We decided against this idea because it poses some localization challenges. For example, the prefix or suffix might change based on the value of the number (e.g. `1 gram` vs `2 grams`). Additionally, there may be cases where units that are a suffix in one locale are a prefix in another (e.g. `56.8 degrees Celsius` in English will be represented as `摂氏 56.8 度` in Japanese). Therefore, we decided to require units to be those supported by ECMAScript so that localization would be built in to the formatter.
```

Comment:
My biggest hesitation is that we don't have an answer for a client that wants to display a non-ECMAScript unit in the cell text (e.g. `5 V`). The closest we have to needing that that I'm aware of are:
1. [specs grid](https://dev.azure.com/ni/DevCentral/_git/Skyline?path=/Web/Workspaces/TestInsights/src/app/specs/component/specs-grid/specs-grid.component.ts) which displays a column for units next to the values. 
2. [tags grid](https://test.lifecyclesolutions.ni.com/tags/) which can display a column for units (%, BYTES) next to the values.

I can easily see someone asking us to display those unit values in the same cell. What should we tell them?

1. If units are uniform across all records:
    1. display units in the column header
    2. write a custom column type for those units that knows how to scale them (e.g. bytes / file size, elapsed time)
2. If units are not uniform across all records:
    1. add a separate column to display units
    2. change to a string column
    3. write a more intelligent custom column type that knows how to scale and sort values that weren't originally uniformly scaled

2.1 and 2.2 would not be sorted correctly, but the legacy grid has the same limitation.

I'm not sure if clients would be satisfied by those options or not. We could try to find out (e.g. chat with Adam A and TJ about these options). Or we could brainstorm more ideas to meet their (possibly imaginary) requirements. Or we could call that out as explicitly not supported for now and wait for them to come to us with a concrete request. But I'm hesitant to do the last option since I feel they've expressed a lot of interest in this topic so I think we should try to satisfy their goals.

Reply:
Maybe we should chat about this again in a post standup discussion. I can't remember where we landed exactly or if we landed somewhere but because tag columns are extra complex (variables types per row) that is separate work and probably something we should design and help implement. I don't remember if specs grid unit requirements are funky or not.

Reply:
Since there are still a lot of open questions about units, we've decided to defer the work to define & implement the API associated with them. I've removed them from the API section of this document and added the alternatives we've discussed to the Alternatives section.